react-components/package.json

```diff
@@ -46,9 +45,9 @@
     "postcss-loader": "2.0.6",
     "promise": "8.0.1",
     "rc-tree": "^1.4.1",
-    "react": "^16.3.2",
+    "react": "^16.4.0",
     "react-dev-utils": "^4.1.0",
-    "react-dom": "^16.2.0",
+    "react-dom": "^16.4.0",
```

Comment:
Should `aphrodite` be removed from package.json or are we still using it elsewhere?

Reply:
`aphrodite` can be removed from package.json